src/main/java/com/cburch/logisim/circuit/appear/AppearancePort.java

```diff
@@ -46,7 +46,7 @@
 import org.w3c.dom.Element;
 
 public class AppearancePort extends AppearanceElement {
-  private static final int INPUT_RADIUS = 4;
+  public static final int INPUT_RADIUS = 4;
```

Comment:
If this is made public, then I think `OUTPUT_RADIUS` and  maybe `MINOR_RADIUS` should be public too for consistency.

Reply:
Yes and no. As they are not used outside the class I would keep them private

Reply:
Of course I do agree that that makes sense, but my point is that if at some point one would need to use it, seeing `INPUT_RADIUS` around, then no IDE won't give any hint about `OUTPUT_RADIUS` due to its visibility. So for consistency, if we expose `INPUT_RADIUS` then I'd expose `OUTPUT_RADIUS` even if not used ATM.

Reply:
Solved it differently by remaking the INPUT_RADIUS private and adding a static function isInputAppearance